esmvaltool/recipes/recipe_quantilebias.yml

```diff
@@ -0,0 +1,84 @@
+# recipe_quantilebias.yml
+---
+
+documentation:
+  description: |
+    Tool for calculation of precipitation quantile bias
+
+  authors:
+    - arno_en
+    - hard_jo
+
```

Comment:
@bouweandela would you like to be the maintainer?
Otherwise, please name someone.

Reply:
It has to be someone from the core development team, right? I'm not sure if I have the time to do this.

Reply:
Maybe @bjoernbroetz ?

Reply:
Actually, it does not need to be someone for the core team (@ruthlorenz for example is the maintainer of her own recipe). So @jhardenberg would be the right person for this, if he agrees (see #785 for details).

Reply:
That would be fine for me, thank you.

Reply:
I've just realized that we forgot to name a maintainer also for the other two recipes you merged this week. Would be OK to put your name in all of those?
Thank you for your help! :handshake:

Reply:
Yes, I realized that too now. It is fine to put me as a maintainer for the ensclus diagnostic. For zmnam instead it would probably be better to name @fserva ...

Reply:
I agree, myself or @chiaracagnazzo can be indicated as maintainer.

Reply:
Thank you! See #785 for more details on maintainer's tasks.